cypress/selectors/mandatees/mandateeSelectors.js

```diff
@@ -1,6 +1,6 @@
 const selectors = {
   mandateeLinkFieldsToggle: '[data-test-mandatee-link-fields-show]',
-  mandateeLinkListItem: '[data-test-mandatee-link-list-item]',
+  mandateeLinkListItem: '[data-test-mandatee-link-agenda-overview-item]',
```

Comment:
Zelfde hier. Deze selector heeft niets te maken met die overview.
'data-test-mandatee-link-list-item' wordt op 2 plaatsen in onze templates gebruikt en gaat dus niet meer werken nu.

Reply:
fixed